src/components/UserInputComponent/GDOGIDMenuComponent.js

```diff
@@ -36,8 +36,15 @@ export default function GDOGIDMenuComponent(props) {
     setOpen(false);
   };
   const handleTooltipOpen = () => {
-    setOpen(true);
+    setOpen((r) => !r);
   };
+  function setChosenGDOGIDFormikValue(gdo_gid) {
+    if (gdo_gid) {
+      formik.setFieldValue("navline.starting_gdo_gid", gdo_gid);
+    } else {
+      formik.setFieldValue("navline.starting_gdo_gid", "");
+    }
+  }
 
   useEffect(() => {
     props.setChosenGDOGIDFormikValue();
```

Comment:
```suggestion
  function setChosenGDOGIDFormikValue(gdo_gid) {
      formik.setFieldValue("navline.starting_gdo_gid", gdo_gid);
  }

  useEffect(() => {
    setChosenGDOGIDFormikValue("");
```

This function could be changed to always set the given parameter to formik. Then, in the useEffect, pass an empty string as an argument.

Reply:
yes but then we should also need to change logic on line 88, to keep consistent types for navline.starting_gdo_gid in formik